packages/radar/src/hooks.ts

```diff
@@ -83,6 +85,11 @@ export const useRadar = <D extends Record<string, unknown>>({
         color: colorByKey[key],
     }))
 
+    const customLegends = legends.map(legend => ({
```

Comment:
What about `boundLegends` instead? those legends aren't necessarily custom, just bound to the data.

Reply:
that's good idea. :)